guides/common/modules/proc_associating-partition-tables-with-disk-encryption.adoc

```diff
@@ -0,0 +1,15 @@
+[id="associating-partition-tables-with-disk-encryption_{context}"]
+= Associating partition tables with disk encryption
+
+{Project} contains partition tables that encrypt the disk of your host using Clevis/Tang during host provisioning.
+You have to associate the partition table with your operating system entry and assign it to your host group or select it manually during provisioning.
```

Comment:
```suggestion
You must associate the partition table with your operating system entry and assign it to your host group or select it manually during provisioning.
```

Reply:
+1 to 'must'.

This sentence is quite long, which makes it harder to scan. 

You might want to break it into two sentences:

You must associate the partition table with your operating system entry. 
Then, you assign it to your host group or select it manually during provisioning.

Reply:
Applied both your suggestions.